presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -117,7 +124,7 @@ public RetryDriver withClassifier(Function<Exception, Exception> classifier)
     /**
      * This is only for sync calls. For Callable that returns Future object, use runAsync instead.
      */
-    public <V> V run(String callableName, Callable<V> callable)
+    public <V> V run(String callableName, RetryStats retryStats, Callable<V> callable)
```

Comment:
Let’s call this `stats`, as the current name is hard to distinguish from `retryStatus`.